docs/installation_c_plus_plus.md

```diff
@@ -0,0 +1,129 @@
+# C++ Installation
+
+!!! note
+      This version of the documentation is for installation of Xailient SDK using wheel file (.whl). 
+      If you are looking for documentation for installion of Xailient SDK using SDK package, please visit our old [documentation](https://xailient.readthedocs.io/en/latest/installation/).
```

Comment:
I'm not sure what "Xailient SDK using SDK package" is referring to. What does "SDK package" mean in this context? I'm not sure readers will understand based on this information alone.

Reply:
Thanks for pointing this out.
This note was added when we migrated from a very old SDK. There was different documentation for it and in order to support users who were still using the old SDK, this note was added. We do not need this now. Hence, removing it.